CesiumGS/cesium
# Inline workers into Cesium.js

Fixes https://github.com/CesiumGS/cesium/issues/11505

Part of https://github.com/CesiumGS/cesium/issues/10619

This inlines all the worker files into `Cesium.js` rather than having `Cesium.js` request each additional worker file. This is appealing because:

* When using `Cesium.js` as a CDN as it reduces the number of additional files requested in turn
* It simplifies deployment for those not using tooling, as the built `Workers` directory does not need to be served, and because cached versions of Cesium.js and the workers will always be the same.
* Theoretically, since network bandwidth will not be used requesting these additional files, time to see content on screen can be reduced.

The CesiumJS build will still ship individual worker modules for those using the ESM version of the library.

| | **Before** | **After** |
| --- | --- | --- | 
| `CesiumUnminified/Cesium.js` | 9.6MB | 12.8MB |
| `CesiumUnminified/Workers` | 2.2MB |  |
| `Cesium/Cesium.js` | 3.8MB | 4.9 MB |
| `Cesium/Workers` | 0.9MB |  |
| `Cesium-1.*.zip` | 68.1MB | 70.2MB |

To test:
* run `npm run make-zip`
* ensure `Build/Cesium/Cesium.js` and `Build/CesiumUnminified/Cesium.js` work
* run `npm start`
* ensure each link on `localhost:8080` still works as expected

TODO:
- [x] Fix specs
- [x] Verify and document built file sizes
